src/qibo/models/circuit.py

```diff
@@ -1099,14 +1099,20 @@ def execute(self, initial_state=None, nshots=1000):
             self._final_state = self.compiled.result(state, nshots)
             return self._final_state
         else:
-            from qibo.backends import GlobalBackend
+            from qibo.backends import _Global
```

Comment:
Did you try lifting the import top-level?

Reply:
Yes. I lifted the import to the top.